index.js

```diff
@@ -621,6 +621,14 @@ function hash(sess) {
  */
 
 function issecure(req, trustProxy) {
+
+  // socket is localhost
+  if (req.connection.remoteAddress === '127.0.0.1' ||
+      req.connection.remoteAddress === '::ffff:127.0.0.1'
+  ) {
```

Comment:
Won't this condition be true for most reverse proxy setups? Or does Express mutate `req.connection.remoteAddress` if `trust proxy` is enabled? (The documentation is not clear.)

Reply:
You are correct. I don't think this will work correctly as any reverse proxy on localhost (think typicall ssl ofloading or even just kubernetes side cars) would end up as thinking this is a localhost connection. Express does not alter any values that are provided from Node.js HTTP core.

Reply:
Yeah I see, I think the reasonable way to do this is to check  if we are behind a proxy